symengine/lib/symengine_wrapper.pyx

```diff
@@ -468,6 +468,131 @@ def get_function_class(function, module):
     return funcs[function]
 
 
+ordering_of_classes = [
```

Comment:
I think in `SymPy` this is used for the relative ordering of different classes within the `Registry`. Again I'm not too sure of the implementation here.